views/editor/nav-options.html

```diff
@@ -112,14 +112,14 @@ <h2 class="snippets-menu-title">{{ gettext("snippetMenuTitle") }}</h2>
         </div>
         <div id="editor-pane-nav-options-menu" class="hide">
           <ul>
-            <li id="editor-option-textsize">
+            <li title="{{ gettext("Change the editor font size") }}" id="editor-option-textsize">
               <span>{{ gettext("navTextSize") }}</span>
               <div class="text-size-ui">
                 <div title="{{ gettext("navDecreaseTextSizeIconTitle") }}" id="editor-pane-nav-decrease-font" class="font-size-button"></div>
                 <div title="{{ gettext("navIncreaseTextSizeIconTitle") }}" id="editor-pane-nav-increase-font" class="font-size-button"></div>
               </div>
             </li>
-            <li id="editor-option-colortheme">
+            <li title="{{ gettext("Change the editor color theme") }}" id="editor-option-colortheme">
```

Comment:
Same thing here, just add a new key/value, maybe `navColorThemeTitle` with this string.